### DISCUSSION TITLE
Can data inside Pandora be easily parsed into JSON?

### DISCUSSION THREAD

**Discussion by Khaligufzel** [bitbrain/pandora#139] [Q&A]
Hi hello!

I'm working on a game where I want other people to easily access/modify/add/remove data without using Godot and Pandora.

So my question is, is there an easy way to store Pandora data in a JSON file?
If not, how would you approach this problem?

Thanks! :)

**Comment by bitbrain** [comment]
@Khaligufzel what is the exact use case? In case you think about modding, worth giving https://github.com/bitbrain/pandora/discussions/80 a read - under the hood Pandora stores data as json, but that is rather an implementation detail. If in future we decide to use a different format, relying on its saving structure could be dangerous and break things.

Pandora is intended to be used with Godot - in case you look for a solution to use something like Pandora in another engine or project outside of Godot, checkout [GridlessDB](https://radmatt.itch.io/gridless).
